src/unix/aix-common.c

```diff
@@ -170,6 +172,10 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
     return UV__ERR(errno);
   }
 
+  if (0 > (sock6fd = socket(AF_INET6, SOCK_DGRAM, IPPROTO_IP))) {
+    return UV__ERR(errno);
```

Comment:
Should `sockfd` be closed here?

Reply:
... Yes it should and I did spot this but somehow haven't pushed the change up to the branch 🤦‍♂.

Reply:
Note to self: `git add ...` before amending/pushing.

Pushed fixup commit.